tests/test_v5_client.py

```diff
@@ -0,0 +1,100 @@
+from luminoso_api.v5_client import LuminosoClient, get_root_url
+from luminoso_api.errors import LuminosoClientError, LuminosoServerError
+
+import pytest
+import requests
+
+BASE_URL = 'http://mock-api.localhost/api/v5/'
+
+
+def test_paths():
```

Comment:
This test appears to be more or less duplicated from the tests in `lumi_stack_tools` -- is there a reason for this?

Reply:
Because I believe it's more appropriate here, having no dependency on the Luminoso stack. And without intervention from Moss, I can't test `lumi_stack_tools`.

Reply:
I'm feeling kind of uneasy about this.  There is already a `test_v5_api_client.py` file in `lumi_stack_tools`, at the top of which is the comment `This file does not actually test code in lumi_stack_tools, but because these tests cannot live in the API client repo itself, this is the next best place.`  I'm inclined to think we should be consistent about where anything meant to test the client lives, and not duplicate tests.

Why is it that you're not able to run the `lumi_stack_tools` tests?  If you were able to run them, would it be satisfactory to move these tests over there?

Reply:
I've described this in the #dev-analytics channel, plus in some one-on-one conversations with Ruth when requested.

I believe that integration tests (making sure the client can use the real API) belong in lumi-stack-tools, and unit tests (testing the specific behavior of the client) belong in the client. Further comments will be outside of this PR.